### PR TITLE
 fix(helmet): Add dashed and missing directives

### PR DESCRIPTION
**Pull request by ffflorian**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/helmetjs/csp/blob/297fd6e511db87885eae9645169547c06b132a6e/lib/config.js https://github.com/helmetjs/csp/blob/297fd6e511db87885eae9645169547c06b132a6e/README.md
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/helmet/helmet-tests.ts [hunk 1]
```diff
@@ -37,14 +38,20 @@ function contentSecurityPolicyTest() {
             frameSrc: emptyArray,
             imgSrc: ['images.example.com'],
             mediaSrc: ['media.example.com'],
+            manifestSrc: ['manifest.example.com'],
             objectSrc: ['objects.example.com'],
             pluginTypes: emptyArray,
+            prefetchSrc: ['prefetch.example.com'],
             reportUri: '/some-url',
-            sandbox: emptyArray,
+            reportTo: '',
```
Comment (EvanHahn) [inline types/helmet/helmet-tests.ts]: Perhaps this should be a real URL instead of the empty string?

**Review by EvanHahn — APPROVED** [review]
👍

I left a small comment but I'm happy with this as is.

**Comment by ffflorian** [comment]
@EvanHahn I added a URL in `reportTo`.

**Review by EvanHahn — APPROVED** [review]

**Comment by ffflorian** [comment]
@horiuchi Sorry, I merged wrong and GitHub automatically requested your review.

Fixed the merge now.

**Comment by ffflorian** [comment]
@EvanHahn Travis had an error. Can you restart the build?

**Comment by EvanHahn** [comment]
@ffflorian I'm not able to restart the build...any way you could push a simple change?

**Comment by ffflorian** [comment]
@EvanHahn It's green :slightly_smiling_face:

**Pull request merged**